pkgs/tools/security/diceware/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib
+, pythonPackages
```

Comment:
Specify `buildPythonApplication` etc. as arguments

Reply:
No, this is not part of `python-packages.nix`, there it should be done. Applications should take `python` or `pythonPackages` as that guarantees packages belong to the same set.

Reply:
Sorry, I didn't know this is the policy.